schema/deploy/shipping/views.sql

```diff
@@ -2088,15 +2088,21 @@ create or replace view shipping.scan_return_results_v1 as
         sample_id,
         barcode as qrcode,
         encountered::date as collect_ts,
-        sample.details @> '{"note": "never-tested"}' as never_tested
+        sample.details @> '{"note": "never-tested"}' as never_tested,
+        sample.details ->> 'swab_type' as swab_type,
+        -- In the future we will want to distinguish between staff-collected
+        -- vs self-collected. Currently all samples that we return results
+        -- on are only self-collected.
+        --  -Jover, 20 July 2020
+        '0' as staff_collected
```

Comment:
Unless there's a good reason to use 0/1 for this flag in https://github.com/seattleflu/lab-result-reports/pull/23, then I think better to use either an actual boolean value or an enumerated/well-known string value.

Reply:
There's no good reason for my choice in seattleflu/lab-result-reports#23 to use 0/1. It was chosen arbitrarily.

Reply:
Should we name this `staff_observed` with values 0/1? 
`observed_by` with values "staff" or "self"?
What are your thoughts here? Trying to have this conversation in one place instead of both here and seattleflu/lab-result-reports#23.

Reply:
`staff_observed` → `{true,false}` seems good to me?

Reply:
Works for me.